python/ray/data/impl/sort.py

```diff
@@ -29,6 +30,8 @@
 from ray.data.impl.push_based_shuffle import PushBasedShufflePlan
 from ray.data.context import DatasetContext
 
+logger = logging.getLogger(__name__)
```

Comment:
Is this logger unused, probably was only used during dev? If so it should be removed.